portfolio.html

```diff
@@ -334,6 +334,7 @@ <h3>LET'S CONNECT</h3>
           Get It Touch
         </button>
       </form>
+      <div class="bottomline"></div>
```

Comment:
Please change the text after the `Contact me` to be `p` tag instead of the `div` tag for semantic purposes.